### PR TITLE
66 table 5

### LINKED ISSUE #66: Add template for table 5

### PR DESCRIPTION
**Pull request by aassuied-ps**
Closes #66

### CODE REVIEW

File: quarto/table-templates/template-table_05.qmd [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+## Function Details
+
+### `make_table_05()`
+
+
+------------------------------------------------------------------------
+
+------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (jknizia) [reply quarto/table-templates/template-table_05.qmd:38]: Okay, now that I compare the sites, I recognize that this shouldn't be removed, but moved to line 44 to seperate the "Required variables" list and the table :-)

**Review by jknizia — CHANGES_REQUESTED** [review]
Hi Alex, 
thanks for the request. I have only one minor comment - other than that it looks very good to me.

File: quarto/table-templates/template-table_05.qmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (aassuied-ps) [reply quarto/table-templates/template-table_05.qmd:38]: Indeed it's working by just deleting one of the horizontal lines and I get the same result as table 6 (horizontal line just above  the table header), is it what you're talking about? I don't see a difference in the code that could explain why table 2 does not have it though...

**Review by jknizia — APPROVED** [review]
Thanks Alex, looks good now!

**Pull request merged**